apps/portals/src/portal-components/DetailsPage/DetailsPage.tsx

```diff
@@ -340,9 +340,7 @@ export const SplitStringToComponent: React.FC<{
   })
   if (overrideSqlSourceTable) {
     // use the search param value to override the sql param.
-    injectedProps['sql'] = `SELECT  *  FROM  ${value}${
-      rowVersionNumber ? `.${rowVersionNumber}` : ''
-    }`
+    injectedProps['sql'] = `SELECT  *  FROM  ${value}`
```

Comment:
Using the row version number with the column value only makes sense if the column value is the ID